From 45ad91efa3c7fa4948bd67f4cacff6dca7e178c2 Mon Sep 17 00:00:00 2001
From: Thierry FOURNIER <tfournier@exceliance.fr>
Date: Wed, 16 Apr 2014 11:10:53 +0200
Subject: [PATCH 140/140] DOC: Update "language" documentation

The example provided with the "language" filter documentation
doesn' run because the match method is not specified.
---
 doc/configuration.txt | 22 ++++++++++------------
 1 file changed, 10 insertions(+), 12 deletions(-)

diff --git a/doc/configuration.txt b/doc/configuration.txt
index a6a391a..5ffcb4f 100644
--- a/doc/configuration.txt
+++ b/doc/configuration.txt
@@ -9427,18 +9427,16 @@ The currently available list of transformation keywords include :
 
                  Example :
 
-                     # this configuration switches to the backend matching a
-                     # given language based on the request :
-
-                     acl de req.fhdr(accept-language),language(de;es;fr;en) de
-                     acl es req.fhdr(accept-language),language(de;es;fr;en) es
-                     acl fr req.fhdr(accept-language),language(de;es;fr;en) fr
-                     acl en req.fhdr(accept-language),language(de;es;fr;en) en
-                     use_backend german  if de
-                     use_backend spanish if es
-                     use_backend french  if fr
-                     use_backend english if en
-                     default_backend choose_your_language
+                   # this configuration switches to the backend matching a
+                   # given language based on the request :
+
+                   acl es req.fhdr(accept-language),language(es;fr;en) -m str es
+                   acl fr req.fhdr(accept-language),language(es;fr;en) -m str fr
+                   acl en req.fhdr(accept-language),language(es;fr;en) -m str en
+                   use_backend spanish if es
+                   use_backend french  if fr
+                   use_backend english if en
+                   default_backend choose_your_language
 
   map(<map_file>[,<default_value>])
   map_<match_type>(<map_file>[,<default_value>])
-- 
1.8.3.2

